include/robot_localization/filter_base.h

```diff
@@ -63,6 +63,9 @@ struct Measurement
   // The time stamp of the most recent control term (needed for lagged data)
   double latestControlTime_;
 
+  // Always log the mahalanobis distance for the particula measurement if the debug flag is on
```

Comment:
```suggestion
  // Always log the mahalanobis distance for the particular measurement if the debug flag is on
```

Reply:
fixed the comment to match updated publish variable.